docs/USAGE.md

```diff
@@ -4,20 +4,31 @@
 
 ## Configuration
 
-## General
+### General
 
 | Environment variable | Description | Default |
 |-|-|-|
-| `OTEL_SERVICE` | Application's default service name. |  |
-| `OTEL_ENV` | The value for the `environment` tag added to every span. |  |
-| `OTEL_VERSION` | The application's version that will populate `version` tag on spans. |  |
-| `OTEL_TAGS` | Comma-separated list of key-value pairs to specify global span tags. For example: `"key1:val1,key2:val2"` |  |
 | `OTEL_TRACE_ENABLED` | Enable to activate the tracer. | `true` |
 | `OTEL_DOTNET_TRACER_FORCE` | Enable the tracer even if no integrations is set using `OTEL_INTEGRATIONS`. | `true` |
 | `OTEL_PROFILER_PROCESSES` | Sets the filename of executables the profiler can attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with comma, for example: `MyApp.exe,dotnet.exe` |  |
 | `OTEL_PROFILER_EXCLUDE_PROCESSES` | Sets the filename of executables the profiler cannot attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with comma, for example: `MyApp.exe,dotnet.exe` |  |
 
-## Instrumentations
+### Resource
+
+Resource is the immutable representation of the entity producing the telemetry.
+See [Resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions)
+for more details.
+
+| Environment variable | Description | Default |
+|-|-|-|
+| `OTEL_RESOURCE_ATTRIBUTES` | Key-value pairs to be used as resource attributes. See [Resource SDK](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md#specifying-resource-information-via-an-environment-variable) for more details. | See [Resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#semantic-attributes-with-sdk-provided-default-value) for details. |
+| `OTEL_SERVICE_NAME` | Sets the value of the [`service.name`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service) resource attribute | | If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence. |
+| `OTEL_SERVICE` | Application's default service name. **Deprected.** |  |
```

Comment:
What "deprecated" means in this case? They don't work anymore? Or they still work but "lose" to resources?

Perhaps it is better to have a separate table for the deprecated ones instead of mixing together with the current ones.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/287 here is the reason.

I did not want to do too much in this PR, however, I wanted to mention that these are not env vars that we want to support. Maybe I should simply remove these rows? Or move to a separate table as you suggest.

Reply:
Removing is fine: we didn't do a release yet so not exposing people to these seems a good option.

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/288/commits/fa1e8497df55061899b269ce6c62037183e187ce